ddpui/utils/email_templates.py

```
@@ -192,3 +192,149 @@ def render_mention_email(
+    # Plain-text version
+    plain_text = (
+        f'{sender_name} has shared "{report_title}" with you\n'
+        f"\n"
+        f"Check it out on Dalgo web for the best experience.\n"
+        f"\n"
+        f"View the report (login required): {private_url}\n"
+    )
+    if public_url:
+        plain_text += f"\nPublic link (no login required): {public_url}\n"
+    plain_text += (
+        f"\n"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)